code/__HELPERS/unsorted.dm

```diff
@@ -540,9 +540,20 @@ Returns 1 if the chain up to the area contains the given typepath
 /proc/between(var/low, var/middle, var/high)
 	return max(min(middle, high), low)
 
-proc/arctan(x)
+
+
+#if DM_VERSION > 513
```

Comment:
Shouldnt this be `>=`? This wont trigger if its 513

Reply:
Edit: its just a byproduct of porting https://github.com/tgstation/tgstation/pull/47137

Reply:
This `#if` is for the warning messgae, the actual code change below is locked to `< 513` since `arctan` exists in 513